Automattic/wp-calypso
# woop: uses combobox to navigate countries

#### Changes proposed in this Pull Request
* Updating Country/Region dropdown to use ComboboxControl

<img width="1058" alt="Screen Shot 2022-01-10 at 2 25 40 PM" src="https://user-images.githubusercontent.com/1689238/148826683-807fded8-39a6-4704-9e59-eb3726fd37bb.png">

#### Testing instructions
* Apply D72720-code and sandbox the API.
* Navigate to `http://calypso.localhost:3000/start/woocommerce-install/store-address?site=<siteId>`
* Select a country (and region, if available) in the country dropdown
* Fill the rest and click continue
* With D72639-code it should send an http request to settings with `woocommerce_default_country` being the code of the selected country.

Related to #
* https://github.com/Automattic/wp-calypso/issues/59833
